test/lint/negated-is-bool.sh

```diff
@@ -2,7 +2,7 @@
 
 echo "Checking usage of negated shared.Is(True|False)*() functions..."
 
-OUT=$(grep -Pr --exclude-dir=.git '!(shared\.)?Is(True|False).*\(' . 2>/dev/null || true)
+OUT=$(git grep -P '!(shared\.)?Is(True|False).*\(' '*.go' || true)
```

Comment:
Does git grep work also on uncommitted files? If not I think we should stop using it for these static analysis checks as it should work on uncommitted files when run locally.